tests/src/core/users/DetermineLifetimeConditions.test.ts

```diff
@@ -325,7 +262,7 @@ describe('Determining Lifetime conditions', () => {
 
       jest.spyOn(tiersService, 'getTierProductsByProductsId').mockResolvedValue(tierFromProduct);
 
-      const result = await determineConditions.getHigherTier([productId], null);
+      const result = await (determineLifetimeConditions as any).getHigherTier([productId], null);
```

Comment:
Why is the any needed always if the return type of the factory is typed?

Reply:
Because they are private functions.

Reply:
How are they private? They are a return of the factory

<img width="463" height="453" alt="image" src="https://github.com/user-attachments/assets/819cf488-2aad-473d-b397-474631d66e93" />

Reply:
I mean, `getHigherTier` is a private function inside `determineLifetimeConditions`

Reply:
I tink I cannot spy `Class['functionKey']`. Maybe the `@ts-ignore` approach could do the work. I'm going to try this way.